### PR TITLE
Merge Overdrive into develop

### PR DESCRIPTION
**Pull request by ErnestFan**
**What's this do?**
This merges Overdrive into latest develop

**Why are we doing this? (w/ JIRA link if applicable)**
Get OD ready to release

**How should this be tested? / Do these changes have associated tests?**
N/A

**Dependencies for merging? Releasing to production?**
Updated the cartfile ref of NYPLAudiobookToolkit and NYPLAEToolkit

**Has the application documentation been updated for these changes?**
N/A

**Did someone actually run this code to verify it works?**
@ErnestFan

### CODE REVIEW

File: Simplified.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2811,7 +2815,8 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				CURRENT_PROJECT_VERSION = 5;
+				CODE_SIGN_STYLE = Manual;
```
Comment (ettore) [inline Simplified.xcodeproj/project.pbxproj:2818]: I think this change should be reverted, we shouldn't change the code signing

File: Simplified.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -2764,7 +2768,7 @@
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Simplified/SimplyE.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
-				CURRENT_PROJECT_VERSION = 5;
+				CURRENT_PROJECT_VERSION = 1;
```
Comment (ettore) [inline Simplified.xcodeproj/project.pbxproj:2771]: can you revert this too? I just changed this

File: Simplified.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ErnestFan) [reply Simplified.xcodeproj/project.pbxproj:2818]: should I revert the `CURRENT_PROJECT_VERSION` on this too?

Comment (ettore) [reply Simplified.xcodeproj/project.pbxproj]: sure

**Pull request merged**